test/Sentry.EntityFramework.Tests/Sentry.EntityFramework.Tests.csproj

```diff
@@ -5,12 +5,12 @@
   </PropertyGroup>
 
   <!-- Running these tests on Mono fail -->
-  <PropertyGroup Condition="'$(OS)' == 'Windows_NT'">
```

Comment:
Here's another case where we were checking incorrectly.  If `OS` isn't defined, then the tests don't run for this target at all on any OS.